plugins/vpp/ifplugin/vppcalls/vpp2001_324/dump_interface_vppcalls.go

```diff
@@ -921,6 +927,58 @@ func (h *InterfaceVppHandler) dumpGreDetails(interfaces map[uint32]*vppcalls.Int
 	return nil
 }
 
+// dumpGtpuDetails dumps GTP-U interface details from VPP and fills them into the provided interface map.
```

Comment:
It looks like `plugins/vpp/ifplugin/vppcalls/vpp2001_324` and `plugins/vpp/ifplugin/vppcalls/vpp2001_379` are almost identical with minor alteration. For these reasons, we have to implement the GTPU changes twice.

Is there any better way? What is Ligato authors' take on this?

We might avoid code duplication by only supporting the later (379) version.

Reply:
I do not like this either, the code is duplicated indeed everywhere, I do not think we can do too much. I guess support for vpp2001_X will be dropped at some point when vpp 20.01 becomes stable and there will be only a vpp2001 subfolder.